src/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="robots" content="noindex, nofollow" />
+    <meta http-equiv="Cache-control" content="no-cache, no-store, must-revalidate">
```

Comment:
seems that `no-store` directive conflicts with no-cache one (see the example [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#preventing_storing)) so maybe here a no-cache with a max-age=0 could be used, what do you think?